integration-cli/docker_api_stats_test.go

```diff
@@ -231,6 +231,9 @@ func (s *DockerSuite) TestApiStatsContainerNotFound(c *check.C) {
 func (s *DockerSuite) TestApiStatsContainerGetMemoryLimit(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 
+	if !isCgroupMounted("memory") {
```

Comment:
Hm, thinking of this; this check should be performed on the *daemon*, not on the client (which may be on a different host and platform). This may become more complicated than I anticipated; we cannot use `SameHostDaemon` as a requirement for the test, because we also want this test to be run on (e.g.) a Windows client against a Linux daemon.

@vdemeester do you have any ideas how to solve that?